### PR TITLE
Test refactor: hex.js

### PR DESCRIPTION
**Pull request by ccowell**
Updated hex.js

Note:
- Moved noLeadingZeros into `security/lib/`

### CODE REVIEW

File: test/security/hex.js [hunk 1]
```diff
@@ -1,148 +1,78 @@
...
-        id: 2
-      };
+    let method = "eth_getTransactionCount";
+    let params = [accounts[0], "pending"];
 
-      provider.sendAsync(request, function() {
-        // Ignore eth_sendTransaction result, it returns the transaction hash.
-        // A transaction hash is a 'DATA' type, which can have leading zeroes
-        // to pad it to an even string length (4 bit per char, so whole bytes).
+    let result = send(method, params, web3);
```
Comment (davidmurdoch) [inline test/security/hex.js]: `send` needs to be `await`ed, right?

Comment (davidmurdoch) [reply test/security/hex.js]: @ccowell 	☝️

Comment (ccowell) [reply test/security/hex.js]: Good catch but we skip this test. Should we keep it?  @davidmurdoch

Comment (davidmurdoch) [reply test/security/hex.js]: Yes, we should keep it.

**Pull request merged**